### DISCUSSION TITLE
Windows images with iconv

### DISCUSSION THREAD

**Discussion by ventaquil** [actions/virtual-environments#5814] [Installed software]
I cannot build software on Windows images (Linux and Mac work fine) because lack of iconv library.

As far as I see it's relatively easy to add iconv on Windows images with NuGet - https://www.nuget.org/packages/libiconv/.

```
PS> Install-Package libiconv -Version 1.14.0.11
```

What do you think?

Relative issue - GH-560.

**Comment by al-cheb** [comment]
We don't see any issues to install iconv library in runtime and build software.
